elm.json

```diff
@@ -5,7 +5,11 @@
     "license": "MIT",
     "version": "1.0.0",
     "exposed-modules": [
-        "CodeGen"
+        "NoDebug.Todo",
```

Comment:
This module doesn't exist. I'm guessing it should be `NoDebug.TodoOrToString`? Also I feel like I'm missing something obvious, why is it called `TodoOrToString`?